pkg/kv/kvserver/replica_raft.go

```diff
@@ -2753,16 +2753,21 @@ func shouldTransferRaftLeadershipToLeaseholderLocked(
 	return lhCaughtUp
 }
 
+type lastReplicaUpdateTime struct {
+	update      time.Time
+	unreachable bool
```

Comment:
Needed to add this field so that `addUnreachableRemoteReplica` / `ReportUnreachable` are called only once per disconnect. Otherwise, these are triggered every tick from `updateLastUpdateTimesUsingStoreLivenessRLocked`.